### DISCUSSION TITLE
Switch languages with key combinations

### DISCUSSION THREAD

**Discussion by random-jaylen** [google/mozc#507] [Q&A]
I need to have many languages on my system, so I don't desire to rely on `Super + Space` to toggle between languages. I would like to mimic Windows and use my own defined key combos to switch between languages.

As an example, on my Windows 10 Pro system, I use `Ctrl + Shift + 1` to switch to Chinese (Bopomofo), `Left Alt + Shift + 1` for Chinese (Pinyin), `Ctrl + Shift + 2` for Korean, `Ctrl + Shift + 3` for English, `Ctrl + Shift + 4` for Japanese, `Ctrl + 1` for Cantonese CPIME, `Ctrl + 2` for Cantonese Cangjie and so on for other languages.

Is this possible with Mozc? 

I checked the Mozc keymap editor, but all the options are really confusing and I don't understand how to implement the above desired settings.

Thank you.

**Comment by hiroyuki-komatsu** [comment]
Hi, this is a feature of IMF (e.g. Fcitx, Ibus), but not IME (e.g. Mozc, Anthy).

So Mozc does not have a functionality to switch to other IMEs.
